### PR TITLE
76810 designfixes

### PR DESCRIPTION
**Pull request by stevelee-civica**
fix back-to-top css
change display name from Telephone number to Phone number

### CODE REVIEW

File: src/EA.Weee.Core/AatfReturn/AatfContactData.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public AatfContactData(Guid id, string firstName, string lastName, string positi
         [Required]
         [StringLength(CommonMaxFieldLengths.Telephone)]
         [Display(Name = "Phone")]
-        [GenericPhoneNumber(ErrorMessage = "The telephone number can use numbers, spaces and some special characters (-+). It must be no longer than 20 characters.")]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Core/AatfReturn/AatfContactData.cs:51]: @stevelee-civica  Dont worry about changing this one. Its a legacy screen so dont need to change.

File: src/EA.Weee.Core/Shared/AddressData.cs [hunk 2]
```diff
@@ -47,7 +47,7 @@ public class AddressData : IValidatableObject
         [Required]
         [StringLength(CommonMaxFieldLengths.Telephone)]
         [Display(Name = "Phone")]
-        [GenericPhoneNumber(ErrorMessage = "The telephone number can use numbers, spaces and some special characters (-+). It must be no longer than 20 characters.")]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Core/Shared/AddressData.cs:50]: @stevelee-civica Dont worry about changing this one. Its a legacy screen so dont need to change.

**Comment by JohnPadgettCivica** [comment]
@stevelee-civica  we also need to add the For International numbers... text as below to the contact details screen the same as the rep contact details.
image

File: src/EA.Weee.Core/AatfReturn/AatfContactData.cs (hunk 1 above)
Comment (stevelee-civica) [reply src/EA.Weee.Core/AatfReturn/AatfContactData.cs]: rolled back

File: src/EA.Weee.Core/Shared/AddressData.cs (hunk 2 above)
Comment (stevelee-civica) [reply src/EA.Weee.Core/Shared/AddressData.cs]: rolled back

File: src/EA.Weee.Web/Views/Shared/_ViewOrganisationDetails.cshtml [hunk 3]
```diff
@@ -15,7 +15,7 @@
                         { "County or region", Model.OrganisationData.BusinessAddress.CountyOrRegion },
```
[the remark this answers is not in the document]
Comment (stevelee-civica) [reply src/EA.Weee.Web/Views/Shared/_ViewOrganisationDetails.cshtml]: rolled back

**Comment by stevelee-civica** [comment]
added phone hint to contact details

**Pull request merged**